src/main/resources/assets/create/lang/zh_cn.json

```
@@ -2384,11 +2873,95 @@
+	"create.ponder.train_assembly.text_13": "列车只能在车站被拆解为方块",
+	"create.ponder.train_assembly.text_14": "在车站上使用时，地图会在该位置添加一个带标签的标记",
+	"create.ponder.train_assembly.text_15": "组装好的列车可以使用扳手重新安置到相邻的轨道",
+	"create.ponder.train_assembly.text_2": "车站是你的轨道网络的路径点",
+	"create.ponder.train_assembly.text_3": "要创建新列车，打开UI并切换到组装模式",
+	"create.ponder.train_assembly.text_4": "在组装期间，时刻表上的列车不会接近这个车站",
+	"create.ponder.train_assembly.text_5": "在轨道上使用列车机壳创建新的转向架",
+	"create.ponder.train_assembly.text_6": "再次单击轨道以在转向架设计之间循环",
+	"create.ponder.train_assembly.text_7": "在强力胶的帮助下连接方块",
+	"create.ponder.train_assembly.text_8": "如果组装好的列车可以在组装好的箱子或桶中找到燃料，它们会移动得更快",
+	"create.ponder.train_assembly.text_9": "储存在保险库中的燃料不会被列车消耗",
+
+	"create.ponder.train_controls.header": "控制列车",
+	"create.ponder.train_controls.text_1": "每个列车装置都需要列车驾驶台",
+	"create.ponder.train_controls.text_2": "组装好后，右击方块开始驾驶",
+	"create.ponder.train_controls.text_3": "使用移动键加速和引导火车",
+	"create.ponder.train_controls.text_4": "如果需要，可以使用鼠标滚轮微调最高速度",
+	"create.ponder.train_controls.text_5": "按住空格键来接近最近的车站",
+	"create.ponder.train_controls.text_6": "列车只能在车站被拆解为方块",
+	"create.ponder.train_controls.text_7": "组装的蒸汽笛可以用冲刺键激活",
+	"create.ponder.train_controls.text_8": "潜行或再次点击可以停止控制列车",
+
+	"create.ponder.train_schedule.header": "使用列车时刻表",
+	"create.ponder.train_schedule.text_1": "时刻表让列车可以被其他司机控制",
+	"create.ponder.train_schedule.text_2": "将物品拿在手上右击打开界面",
+	"create.ponder.train_schedule.text_3": "一旦规划好，时刻表就可以交给列车司机",
+	"create.ponder.train_schedule.text_4": "任何坐在列车驾驶台前面的生物或烈焰人燃烧室都是合格的列车长",
+	"create.ponder.train_schedule.text_5": "用拴绳拴住生物可以更方便地给予它们座位",
+	"create.ponder.train_schedule.text_6": "可以随时从司机那里取回时刻表",
+
+	"create.ponder.train_signal_placement.header": "放置列车信号机",
+	"create.ponder.train_signal_placement.text_1": "选择一个列车轨道，然后将信号机放置在附近",
+	"create.ponder.train_signal_placement.text_2": "信号机控制不由玩家驾驶的列车的流动",
+	"create.ponder.train_signal_placement.text_3": "被调度的列车绝不会以相反方向穿过信号",
+	"create.ponder.train_signal_placement.text_4": "……除非添加第二个面向相反方向的信号",
+	"create.ponder.train_signal_placement.text_5": "可以连接辉光管让信号灯更加显眼",
+
+	"create.ponder.train_signal_redstone.header": "信号机和红石",
+	"create.ponder.train_signal_redstone.text_1": "信号机通入红石信号后可以保持为红色",
+	"create.ponder.train_signal_redstone.text_2": "相反，红色信号产生比较器输出",
+
+	"create.ponder.train_signal_signaling.header": "用信号机避免相撞",
+	"create.ponder.train_signal_signaling.text_1": "列车信号机将轨道分成多个区间",
+	"create.ponder.train_signal_signaling.text_2": "如果一个区间被占用，则不允许其他列车进入",
+	"create.ponder.train_signal_signaling.text_3": "因此，每个区间同时只能容纳一列列车",
+	"create.ponder.train_signal_signaling.text_4": "通过扳手可以使用第二种信号模式",
+	"create.ponder.train_signal_signaling.text_5": "黄铜信号区间通常会产生标准信号",
+	"create.ponder.train_signal_signaling.text_6": "这种特殊的信号可以在第二种情况下停止列车",
+	"create.ponder.train_signal_signaling.text_7": "它会停止列车，该列车在进入时……",
+	"create.ponder.train_signal_signaling.text_8": "……将无法立即离开该区间",
+	"create.ponder.train_signal_signaling.text_9": "这有助于排队的列车远离繁忙的区间",
```

```suggestion
	"create.ponder.train_signal_signaling.text_9": "这有助于等候的列车远离繁忙的区间",
```

---

原文是

> This helps keeping queued Trains out of a busy Segment

再根据上下文和涉及的情景，应该是包含“排队”（会导致区间被占用）和“等候”（等红灯，会导致一段时间内区间占用情况不会变化）的意思的，所以应该综合起来“排队等候”

```suggestion
	"create.ponder.train_signal_signaling.text_9": "这有助于排队等候的列车远离繁忙的区间",
```